UPGRADE.md

```diff
@@ -1,5 +1,12 @@
 # Upgrade documentation
 
+## Upgrade from 2.6.1 or lower
+
+Database schema changed and needs to be updated:
+
+    ALTER TABLE virtual_users
+    ADD wkd_key LONGTEXT DEFAULT NULL;
```

Comment:
You changed the format of the key to binary. Would it make sense to also adjust the database field type?

Reply:
I decided to go with base64 encoded binary instead to not have binary blobs in the database. Would you prefer binary blobs in database? If so, why?